doc: add meeting minutes Aug 8 (#1141)
openjs-foundation/cross-project-council: meetings/2023/2023-08-08.md

# OpenJS Foundation Cross Project Council Meeting 2023-08-08

## Links

> * **Recording**: 

Author: ```suggestion
* **Recording**: https://www.youtube.com/watch?v=BZnz21hQ83E
```

* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1132

## Present

* Joe Sepi (@joesepi)
* Ben (@bensternthal)
* Rick Markins (@rxmarbles)
* Claudio Wunder (@ovflowd)
* Sean Larkin (@thelarkinn)
* Rich Trott (@Trott)
* Tobie Langel (@tobie)
* Christian Bromann (@christian-bromann)
* Adrian Estrada (@edsadr)
* Robin Ginn (@rginn)
* Tierney Cyren (@bnb)
* Sara Chipps (@SaraJo)
* Kylie Wagar-Dirks (@kyliewd) 

## Agenda

### Announcements

* Claudio - nodejs collab summit registration is live
https://github.com/openjs-foundation/summit/issues/360

### Board Meeting Updates

> *Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.

Reviewer: ```suggestion
* Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```

### openjs-foundation/cross-project-council

* Upcoming CPC Elections [#1126](https://github.com/openjs-foundation/cross-project-council/issues/1126)

>   * Ben work with Joe on elections, ensure nomination respondes are public (Tobie request)

Reviewer: ```suggestion
  * Ben work with Joe on elections, ensure nomination responses are public (Tobie's request)
```

* Code of Conduct Plan of Action [#1122](https://github.com/openjs-foundation/cross-project-council/issues/1122)
  * progress is happening; take a look at open PRs; any help is appreciated

* add @edsadr as a regular member to CPC [#1121](https://github.com/openjs-foundation/cross-project-council/pull/1121)

>   * Welcome to Adrian; as end user director

Reviewer: ```suggestion
  * Welcome to Adrian; as end-user director
```

Reviewer 2: The actual title is End User Director (without the hyphen), so let's leave it as such.

  * PR Merged

* Standardize on Collab Spaces over Working Groups [#1110](https://github.com/openjs-foundation/cross-project-council/issues/1110)
  * Joe working on it
  * Tobie, double checking on discussion with standard working group
  * Revisit code of conduct working group, decision made to close since this is now part of CPC

* Non-violent communication / strategic workplace conflict resolution training? [#1106](https://github.com/openjs-foundation/cross-project-council/issues/1106)
  * Robin - next step is to go back to LF HR and see what it would take to put together a training for us, will also check on previous training (that Tracy had helped organize)

> * Enable ability to have pseudonimity for EasyCLA. [#1088](https://github.com/openjs-foundation/cross-project-council/issues/1088)

Reviewer: ```suggestion
* Enable the ability to have pseudonymity for EasyCLA. [#1088](https://github.com/openjs-foundation/cross-project-council/issues/1088)
```

Reviewer 2: That's the actual title of the issue, so let's not fix it here.
